src/components/events/Event.js

```diff
@@ -1,7 +1,34 @@
+import Link from "next/link";
 import React from "react";
-
-const Event = () => {
-  return <div>Event</div>;
+import { FiExternalLink } from "react-icons/fi";
+const Event = ({
+  month = "FEB",
+  day = "29",
+  start = "7:00PM",
+  title = "Life in the Industry: PM",
+  location = "http://google.com",
+  description = "For our second installment of the Life in the Industry series: Learn all about the role as a Product Manager, from the details of the day-to-day work schedule to the ranking status, salaries, and the people you work with in this position.",
+  color = "bg-winc-pink",
+}) => {
+  return (
+    <div className="font-urbanist flex rounded-2xl overflow-hidden max-w-5xl m-40">
+      <div className={`${color} text-winc-white text-center w-60 h-60`}>
```

Comment:
look at the reference code, there is no hard coded width or height values. Make it based on how much percentage of width it should take up.